pkg/reconciler/v1alpha1/pipelinerun/pipelinerun.go

```diff
@@ -176,22 +176,25 @@ func (c *Reconciler) Reconcile(ctx context.Context, key string) error {
 
 	if pr.IsDone() {
 		c.timeoutHandler.Release(pr)
-		c.Recorder.Event(pr, corev1.EventTypeNormal, eventReasonSucceeded, "PipelineRun completed successfully.")
```

Comment:
hmm, looks like in this change this line got removed, so we no longer emit an event indicating successful completion? :thinking:

Reply:
Oops, will submit a followup pr to fix that.

Reply:
:pray: thanks!!